pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```diff
@@ -76,11 +86,34 @@ func (k *KubeASCheck) Configure(config, initConfig integration.Data, source stri
 		log.Error("could not parse the config for the API server")
 		return err
 	}
-
-	log.Debugf("Running config %s", config)
+	if k.instance.CardinalityLimit == 0 {
+		k.instance.CardinalityLimit = 200
+	}
+	k.ignoredEvents = convertFilter(k.instance.FilteredEventTypes)
 	return nil
 }
 
+func convertFilter(conf []string) string {
```

Comment:
This is just to avoid a breaking change, before users could just use the reason value (e.g. `CrashLoopBackOff` and I wanted to extend this to support kind and type as well.
Now users can specify `type:Normal,kind:Pod` for instance. I find this approach more transparent and user friendly than having the user input `type!=Normal,involvedObject.kind!=Pod` which is more error prone in my opinion

Reply:
Okay